tests/shadow_price.rs

```diff
@@ -31,13 +32,19 @@ mod tests {
             .solve()
             .expect("Library test");
 
-        assert_eq!(solution.eval(&objective), 77.30220492866408);
-        assert_eq!(solution.value(x1), 20.363164721141374);
-        assert_eq!(solution.value(x2), 6.485084306095981);
-        assert_eq!(solution.get_dual_values()[3], solution.get_dual_value(3));
-        assert_eq!(
-            solution.get_dual_values(),
-            vec![-0.0, -0.0, -0.6809338521400778, -0.09208819714656294]
+        assert_float_eq!(77.30220492866408, solution.eval(&objective), abs <= 1e-10);
+        assert_float_eq!(20.363164721141374, solution.value(x1), abs <= 1e-10);
+        assert_float_eq!(6.485084306095981, solution.value(x2), abs <= 1e-10);
```

Comment:
Can we choose a problem with more "round" solutions ? These are easier to check.